plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -78,57 +78,58 @@ object ScalafmtPlugin extends AutoPlugin {
 
   val globalInstance = Scalafmt.create(this.getClass.getClassLoader)
 
+  private def withMaybePlural(count: Int, singular: String) =
+    s"$count $singular" + (if (count > 1) "s" else "")
```

Comment:
In my opinion mixing string interpolation with concatenation is not good practice. This line can be rewritten more consistent:

```scala
s"$count $singular${if (count > 1) "s" else ""}"
```